src/components/header/Header.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+const HeaderContainer = styled.div`
+  background-color: #e4e4ee;
+  display: flex;
+  justify-content: center;
+  padding: 10px;
+  h1 {
+    font-weight: lighter;
+    font-size: 25px;
+  }
+  .fa-bars {
+    position: absolute;
```

Comment:
It would be better to use the flex we're inside here to position these rather than making them all absolute. As a rule of thumb `position: absolute` is a last resort.

Reply:
Done!